packages/core-typings/src/IMessage/IMessage.ts

```diff
@@ -124,6 +124,10 @@ export type MessageMention = {
 	_id: string;
 	name?: string;
 	username?: string;
+<<<<<<< Updated upstream
+=======
+	fname?: string; // incase of channel mentions
+>>>>>>> Stashed changes
```

Comment:
@yash-rajpal I think this might be a mistake. It would cause build to fail.

Reply:
Ooops, how did it get there 😓